lib/widgets/shared/problem_page.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:crowdleague/models/problems/problem_base.dart';
+import 'package:flutter/material.dart';
+
+class ProblemPage extends StatelessWidget {
```

Comment:
Any interest in dropping a really brief class description comment at the top?  Class just seems kinda naked without one :-)  I am aware of the hypocrisy but I look forward to being asked to put some clothes on my classes!